modules/nf-core/fqtk/main.nf

```diff
@@ -0,0 +1,47 @@
+process FQTK {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::fqtk=0.2.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/fqtk:0.2.0--h9f5acd7_0' :
+        'quay.io/biocontainers/fqtk:0.2.0--h9f5acd7_0' }"
+
+    input:
+    tuple val(meta), path(sample_sheet), path(fastq_dir)
```

Comment:
`fastq_dir` is not used, can you explain?

Reply:
This is a rough draft, and it was just in trying to keep with the format of demultiplex. Likely not necessary once I re-work the input.

Reply:
Given this module doesn't have to be used with the demultiplex pipeline, lets do the simplest thing.